students/ws/users/views.py

```diff
@@ -38,24 +40,36 @@ def post(self, request):
             )
 
             return JsonResponse({"message": "SIGHUP SUCCESS"}, status=201)
+
         except KeyError:
             return JsonResponse({"message" : "KEYERROR"}, status = 400)
+
         except ValidationError as error:
             return JsonResponse({"message" : error.message}, status = 400)
 
 class LogInView(View):
     def post(self, request):
         try:
             data            = json.loads(request.body)
+
             email_insert    = data['email']
             password_insert = data['password']
 
             if not User.objects.filter(email = email_insert).exists():
                 return JsonResponse({"message" : "INVALID_USER"}, status = 401)
 
-            if User.objects.get(email = email_insert).password != password_insert:
+            password_db_encoded     = User.objects.get(email = email_insert).password.encode('utf-8')
+            password_insert_encoded = password_insert.encode('utf-8')
+
+            if not bcrypt.checkpw(password_insert_encoded, password_db_encoded):
                 return JsonResponse({"message" : "INVALID_USER"}, status = 401)
 
-            return JsonResponse({"message": "LOGIN SUCCESS"}, status=200)
+            access_token = jwt.encode({"id" : User.objects.get(email = email_insert).id}, SECRET_KEY, algorithm = ALGORITHM)
```

Comment:
user 라는 변수에 db에서 가져온 data를 담아 놓고 사용하면 매번 ORM으로 요청하지 않아도 됩니다.
```suggestion
            access_token = jwt.encode({"id" : user.id}, SECRET_KEY, algorithm = ALGORITHM)
```